examples/10.3/README.md

```diff
@@ -22,8 +22,8 @@ Verification commands
 Run the following commands to validate things are rolling as they should.
 
 ```bash
-# Should use 10.3.27 as the default version
-lando ssh -s defaults -c "mysql -V | grep 10.3.27"
+# Should use 10.3.39 as the default version
+lando ssh -s defaults -c "mysql -V | grep 10.3.39"
```

Comment:
Change `mysql` to `mariadb`?

Reply:
`mariadb` binaries/symlinks are [available only on 10.4+](https://jira.mariadb.org/browse/MDEV-21303)